runtime/src/builtins.rs

```diff
@@ -231,3 +231,27 @@ pub(crate) fn get() -> Builtins {
         feature_transitions: builtin_feature_transitions(),
     }
 }
+
+pub fn get_pubkeys() -> Vec<Pubkey> {
```

Comment:
a comment /// may help here. I'm not too familiar with all the built ins so some context or summary may be helpful to me and others.